src/index.html

```diff
@@ -22,6 +22,421 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header>
+        <div class="moyo_logo">
+          <a
+            href="index.html"
+            class="logo"
+          >
+            <img
+              src="images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <div>
+          <nav class="nav">
+            <ul class="nav_list">
+              <li class="nav__item">
+                <a
+                  href="Apple"
+                  class="is-active nav_link"
```

Comment:
Fix class everywhere
```suggestion
            <ul class="nav__list">
              <li class="nav__item">
                <a
                  href="Apple"
                  class="is-active nav__link"
```